src/js/edu-benefits/edu-benefits-entry.jsx

```diff
@@ -12,7 +13,7 @@ import routes from './routes.jsx';
 
 require('../../sass/edu-benefits.scss');
 
-const store = createStore(reducer);
+const store = createStore(reducer, compose(applyMiddleware(thunk), __BUILDTYPE__ === 'development' && window.devToolsExtension ? window.devToolsExtension() : undefined));
```

Comment:
It adds two middlewares to Redux. One is [redux-thunk](https://github.com/gaearon/redux-thunk), which lets you return a function in action creators. Rx is using it for api calls, but it also lets you get info from the redux state in the action creator function (you can see it the actions/index.js change I made). It reduces some boilerplate.

The other is to allow the Redux dev tools Chrome extension to read the Redux state. Those tools are pretty useful, if you haven't tried them out: https://chrome.google.com/webstore/detail/redux-devtools/lmhkpmbekcpmknklioeibfkpmmfibljd